tests/litestar/test_litestar.py

```diff
@@ -539,9 +539,9 @@ async def test_optional_session(driver_config_client: TestClient[Litestar]):
         url="handle-session-optional",
     )
 
-    dict_response = json.loads(response.text)
+    Dict_response = json.loads(response.text)
     assert response.status_code == 200
-    assert dict_response["s"] == "empty session"
+    assert Dict_response["s"] == "empty session"
```

Comment:
Unintentional refactor?

Reply:
Yes this was unintentional refactor will change all this.